src/main/java/com/rocketden/main/dto/game/GameTimerDto.java

```diff
@@ -10,8 +14,15 @@
 @Setter
 @EqualsAndHashCode
 public class GameTimerDto {
+
+    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonSerialize(using = LocalDateTimeSerializer.class)
     private LocalDateTime startTime;
-    private Long duration;
+
+    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonSerialize(using = LocalDateTimeSerializer.class)
```

Comment:
What do these fields do?

Reply:
The GameSocketTests had issues converting messages to Java objects without these to help it serialize and deserialize the dates (I guess it has something similar to do with the `toObjectLocalDateTime` method you implemented in an earlier PR)

Not sure if this actually affects the behavior on the frontend (since it seemed like it worked fine in a non-test environment)

Reply:
Ah nice, I wonder if I could've just used that instead of the `toObjectLocalDateTime` method, as this is a lot simpler -- either way, sounds good.